std_execution.bs

```diff
@@ -5161,197 +5161,92 @@ are all well-formed.
-
-2. The name `execution::let_stopped` denotes a customization point object. For some subexpressions `s` and `f`, let `S` be `decltype((s))`, let `F` be the decayed type of `f`, and let `f'` be an xvalue that refers to an object decay-copied from `f`. If `S` does not satisfy `execution::sender` or if `F` does not satisfy `invocable`, `execution::let_stopped` is ill-formed. Otherwise, the expression
-    `execution::let_stopped(s, f)` is expression-equivalent to:
-
-    1. `tag_invoke(execution::let_stopped, get_completion_scheduler<set_value_t>(s), s, f)`, if that expression is valid.
-
-        * <i>Mandates:</i> The type of the `tag_invoke` expression above satisfies `execution::sender`.
-
-    2. Otherwise, `tag_invoke(execution::let_stopped, s, f)`, if that expression is valid.
+                1. For `execution::let_value`, let `base_val_sigs...` be an empty set of types; otherwise let `base_val_sigs...` be <code>value_types_of_t&lt;S, E, <i>set-value-signature</i>, <i>type-list</i>></code>.
 
-        * <i>Mandates:</i> The type of the `tag_invoke` expression above satisfies `execution::sender`.
+                2. For `execution::let_error`, let `base_err_sigs...` be an empty set of types; otherwise let `base_errs...` be <code>error_types_of_t&lt;S, E, <i>type-list</i>></code> and `base_err_sigs...` be `set_error_t(base_errrs)...`.
 
-    3. Otherwise, given a receiver `out_r` and an lvalue `out_r'` refering to an object decay-copied from `out_r`.
-    
-        1. Let `r` be an rvalue of a receiver type `R` such that:
-    
-            1. `execution::set_value(r, args...)` is expression-equivalent to `execution::set_value(std::move(out_r'), args...)`.
+                3. For `execution::let_stopped` or if the expression <code>completion_signatures_of_t&lt;S, E>::sends_stopped</code> is `false`, let `base_stopped_sigs...` be an empty set of types; otherwise let `base_stopped_sigs...` be the set of one type `[set_stopped_t()]`.
 
-            2. `execution::set_error(r, e)` is expression-equivalent to `execution::set_value(std::move(out_r'), e)`.
+                4. For each type <code>S3<i><sub>i</sub></i></code> in `S3s...`, let <code>Vs<i><sub>i</sub></i>...</code> be the list of types in the <code><i>type-list</i></code> named by <code>value_types_of_t&lt;S3<i><sub>i</sub></i>, E, <i>set-value-signature</i>, <i>type-list</i>></code>, let <code>Es<i><sub>i</sub></i>...</code> be the list of types in the <code><i>type-list</i></code> named by <code>error_types_of_t&lt;S3<i><sub>i</sub></i>, E, <i>type-list</i>></code>, let <code>Ds<i><sub>i</sub></i>...</code> be the list of values <code>completion_signatures_of_t&lt;S3s, E>::sends_stopped...</code>, and let <code>S3_sigs<i><sub>i</sub></i></code> be the list of types <code>[Vs<i><sub>i</sub></i>..., Es<i><sub>i</sub></i>..., Ds<i><sub>i</sub></i>...]</code>.
```

Comment:
```suggestion
                4. For each type <code>S3<i><sub>i</sub></i></code> in `S3s...`, let <code>Vs<i><sub>i</sub></i>...</code> be the list of types in the <code><i>type-list</i></code> named by <code>value_types_of_t&lt;S3<i><sub>i</sub></i>, E, <i>set-value-signature</i>, <i>type-list</i>></code>, let <code>Es<i><sub>i</sub></i>...</code> be the list of types in the <code><i>type-list</i></code> named by <code>error_types_of_t&lt;S3<i><sub>i</sub></i>, E, <i>error-result-types</i>></code>, let <code>Ds<i><sub>i</sub></i>...</code> be the list of values <code>completion_signatures_of_t&lt;S3s, E>::sends_stopped...</code>, and let <code>S3_sigs<i><sub>i</sub></i></code> be the list of types <code>[Vs<i><sub>i</sub></i>..., Es<i><sub>i</sub></i>..., Ds<i><sub>i</sub></i>...]</code>.
```

Reply:
I'm not sure I understand this change. Please note that:
- it was like that in the previous version; see [let_value](https://brycelelbach.github.io/wg21_p2300_std_execution/std_execution.html#spec-execution.senders.adapt.let_value), bullet 2.3.3
- `error-result-types` is not defined; defining it different from `type-list` doesn't make sense to me

Reply:
please see my new form. This whole block is completely replaces by a simpler wording. Thank you for pushing me in this direction.